### DISCUSSION TITLE
怎么更改alist默认视频播放器啊

### DISCUSSION THREAD

**Discussion by axixa120** [AlistGo/alist#3041] [Q&A]
怎么更改alist默认视频播放器啊

**Comment by NovaShen555** [comment]
你去看看有个discussion是添加epub浏览器的，可以参考一下

**Comment by anwen-anyi** [comment]
你是指的这样？
![video](http://gchat.qpic.cn/gchatpic_new/0/0-0-1B8579BF0209A3C08E3BD8F4DD5AC563/0)

**Reply by NovaShen555** [reply]
是的，然后这是个frame，你去搭个服务或者什么之类的 就好了

**Reply by anwen-anyi** [reply]
> 是的，然后这是个frame，你去搭个服务或者什么之类的 就好了

是的，自己搭一个在线播放服就行 哈哈

**Reply by STEVENTAN100** [reply]
请问怎么在线网站的链接怎么寻找呢？求大佬指点，直接嵌入

**Reply by NovaShen555** [reply]
蛤，这个直接用官方写好的不就行了吗，你播放不了？

**Reply by STEVENTAN100** [reply]
有些视频播放黑屏有声音，不过感觉应该是10bit视频不能直接浏览器播放吧

**Reply by NovaShen555** [reply]
sor，这个我了解的不是很深入，没办法回答您，您再看看吧

**Reply by STEVENTAN100** [reply]
我发一个链接给您，请问能帮我简单判断一下吗？手机上播放有画面，电脑浏览器播放没有

**Reply by STEVENTAN100** [reply]
以游客方式浏览就好

**Reply by STEVENTAN100** [reply]
ffmpeg -i "要转换的视频文件名" -pix_fmt yuv420p "输出的文件名"，我在目录下使用这个命令转换了一下文件（01.mp4），浏览器可以播放了，是不是可以确定是10bit的问题呢？因为这个问题确实比较小众些

**Reply by NovaShen555** [reply]
这个真不清楚，我试了下，浏览器放不了的，用外部播放器就可以播放，真的帮不了您

**Reply by STEVENTAN100** [reply]
是的呢，我也很奇怪，等高人指点下，谢谢您！

**Reply by 12189108** [reply]
https://alist.nn.ci/faq/why.html#why-can-t-i-preview-the-video-or-audio-file
